lib/Dialect/TritonGPU/Transforms/Pipeliner/WGMMAPipeline.cpp

```diff
@@ -30,6 +33,30 @@ namespace tt = mlir::triton;
 namespace ttg = mlir::triton::gpu;
 namespace ttng = mlir::triton::nvidia_gpu;
 
+// Returns whether the dot dot such that:
+// 1. The LHS comes from registers and
+// 1.1  The LHS is defined inside the loop
```

Comment:
This condition is not reflected in the function name

Reply:
yeah, I didn't have a good name for the function. any proposal?

Reply:
naming is difficult sometimes :)

My potentially bad proposal is `isRSDotFromSIMDInLoop`.... Feel free to ignore

Reply:
changed to rsDotNeedsWait.